client/__tests__/e2e/cellxgeneActions.ts

```diff
@@ -141,6 +141,43 @@ export async function getAllCategoriesAndCounts(
   return Object.fromEntries(arrayOfLabelsAndCounts);
 }
 
+export async function expandMarkerGeneSetsHeader(page: Page): Promise<void> {
+  // Locate and expand the 'Marker Gene Sets' header if not already expanded
+  const markerGeneSetsHeader = await page.locator(
+    "h5:has-text('Marker Gene Sets')"
+  );
+  const chevronDownIcon = markerGeneSetsHeader.locator(
+    "svg[data-icon='chevron-down']"
+  );
+  if ((await chevronDownIcon.count()) === 0) {
+    await markerGeneSetsHeader.click();
```

Comment:
I see that the assertion count > 0 got removed between the latest commit and the one before. Why is that? Thank you!

Reply:
https://github.com/chanzuckerberg/single-cell-explorer/pull/902#discussion_r1583500316

Reply:
Oh, I just realized why the tryUntil wasn't working, lol. Had to await it.

Reply:
Whoopsie

Reply:
Reverted the commit